spec/spec_helper.rb

```diff
@@ -2,3 +2,11 @@
   require 'simplecov'
   SimpleCov.start
 end
+
+# Configure VCR cassettes
+# VCR.configure do |config|
+#   config.ignore_hosts 'codeclimate.com' if ENV['CI']
+#   config.cassette_library_dir = File.join(ManageIQ::Providers::Lenovo::Engine.root, 'spec/vcr_cassettes')
+# end
+
+Dir[Rails.root.join("spec/shared/**/*.rb")].each { |f| require f }
```

Comment:
`Rails.root` in this case is the dummy app `spec/manageiq` and `spec/shared` is where we put all shared examples to be used across provider repos - like introduced in https://github.com/ManageIQ/manageiq/pull/11414 

I think we dont include all `spec/support` from manageiq yet, as we hope to get along without it (reducing dependencies)

Maybe `spec/shared` is badly named - I see it as part of the provider certification part explained in https://github.com/ManageIQ/manageiq/pull/11414